nautobot/core/api/utils.py

```diff
@@ -103,20 +104,20 @@ def get_serializer_for_model(model, prefix=""):
     Raises:
         SerializerNotFound: if the requested serializer cannot be located.
     """
-    app_name, model_name = model._meta.label.split(".")
-    if app_name == "contenttypes" and model_name == "ContentType":
-        app_name = "extras"
+    app_label, model_name = model._meta.label.split(".")
+    if app_label == "contenttypes" and model_name == "ContentType":
+        app_path = "nautobot.extras"
     # Serializers for Django's auth models are in the users app
-    if app_name == "auth":
-        app_name = "users"
-    serializer_name = f"{app_name}.api.serializers.{prefix}{model_name}Serializer"
-    if app_name not in settings.PLUGINS:
-        serializer_name = f"nautobot.{serializer_name}"
+    elif app_label == "auth":
+        app_path = "nautobot.users"
```

Comment:
I prefer not to use this kind of conditions in the code, as it needs future care. Implementation using `__module__` was more generic. Not sure, whether this works with e.g. `Group`:

```
invoke nbshell

In [1]: Group._meta.label.split('.')
Out[1]: ['auth', 'Group']

In [2]: Group.__module__
Out[2]: 'django.contrib.auth.models'
```